flow/packet-headers/icmpv6.yaml

```diff
@@ -0,0 +1,43 @@
+components:
+  schemas:
+    Flow.Icmpv6:
+      description: ICMPv6 packet header
+      type: object
+      properties:
+        type:
+          x-field-pattern:
+            x-constants:
+              destination_unreachable: 1
+              packet_too_big: 2
+              time_exceeded: 3
+              parameter_problem: 4
+              echo_reply: 129
+              echo_request: 128
+              multicast_listener_query: 130
+            description: >-
+              ICMPv6 Type
+            format: integer
+            length: 8
+            default: 4
```

Comment:
is this the correct default?

Reply:
Sorry this should be 1 i.e. destination unreachable. I modified it.

Reply:
I think it should default to `echo_request`

Reply:
Modified it.